kernel/public/test-scenes/-101.105.avatarshape/game.ts

```diff
@@ -0,0 +1,43 @@
+import { engine, Entity, Transform, Vector3, OnPointerDown, ActionButton, BoxShape
+  , executeTask
+  // , Vector2, Material, PlaneShape, Quaternion, TransparencyMode, Texture
```

Comment:
commented code shall be removed

Reply:
Removed that test scene as it wasn't finished and we didn't need it in the end